### PR TITLE
Update to Kubernetes v1.8.5

### PR DESCRIPTION
**Pull request by mikkeloscar**
**Work in progress**

Changelogs:
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v180
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v181
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v182
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v183
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v184
* https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#v185

### CODE REVIEW

File: cluster/userdata-master.yaml [hunk 1]
```diff
@@ -192,7 +191,7 @@ coreos:
           --volume=kube,kind=host,source=/etc/kubernetes,readOnly=true \
           --mount=volume=kube,target=/etc/kubernetes \
           --net=host \
-          docker://registry.opensource.zalan.do/teapot/hyperkube:v1.7.9_coreos.0 \
+          docker://gcr.io/google_containers/hyperkube:v1.8.4 \
```
Comment (Raffo) [inline cluster/userdata-master.yaml]: gcr?

Comment (mikkeloscar) [reply cluster/userdata-master.yaml]: *work in progress*, but thanks for the reminder because I could have totally forgot to change it :)

**Comment by mikkeloscar** [comment]
:+1:

**Comment by Raffo** [comment]
👍

**Pull request merged**